document-store/src/main/java/org/hypertrace/core/documentstore/mongo/MongoCollection.java

```diff
@@ -218,6 +220,12 @@ private String decodeKey(String key) {
     if (query.getFilter() != null) {
       map = parseQuery(query.getFilter());
     }
+
+    Bson projection = new BasicDBObject();
```

Comment:
Please add a github issue to track adding support for the same in PostgresCollection

Reply:
I am assuming that it won't break any functionality on upgrade if it's not implemented Postgres immediately.

Reply:
yes. thanks for reminding me that